src/utils/tree/removeNode/removeNode.ts

```
@@ -64,6 +64,29 @@ const getParentTreeNode = (tree: TreeNode, path: string, uuid: string) => {
   return findParentNodeByUUIDInTree(currentTree, uuid);
 };
 
+const removeAncestorReferences = (node: TreeNode, uuidToRemove: string): TreeNode => {
+  const updatedAttributes = { ...node.attributes };
+
+  if (updatedAttributes.defaultValueFromAncestor?.uuid === uuidToRemove) {
+    delete updatedAttributes.defaultValueFromAncestor;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)